src/main/c/Jep/convert_p2j.c

```diff
@@ -865,6 +865,16 @@ jobject PyObject_As_jobject(JNIEnv *env, PyObject *pyobject,
         if ((*env)->IsAssignableFrom(env, pyjobject->clazz, expectedType)) {
             return (*env)->NewLocalRef(env, pyjobject->object);
         }
+    } else if (PyCallable_Check(pyobject)) {
```

Comment:
Moved up the `PyCallable` check, since it needs to happen before `PyObject` test (since `PyCallable` extends `PyObject`).